src/pl/devsmentoring/java/Module5/Exercise20WithMethods.java

```diff
@@ -0,0 +1,49 @@
+package pl.devsmentoring.java.Module5;
+
+import java.util.*;
+
+public class Exercise20WithMethods {
+    public static void main(String[] args) {
+
+        List<String> colourList = new ArrayList<>(Arrays.asList("zielony", "czerwony", "niebieski", "czarny",
+                "fioletowy", "granatowy", "niebieski",
+                "czarny", "czarny", "zielony", "cytrynowy", "granatowy",
+                "niebieski", "indygo", "zielony", "czerwony"));
+
+        colourCopy(colourList);
+        System.out.println("In set we have " + colourSet.size() + " different colors.\n");
+        displayColorsInNewLines(colourSet);
+        System.out.println("");
+
+        colorAdd(colourSet);
+        System.out.println("");
+        colorRemove(colourSet);
+    }
+
+    private static HashSet<String> colourSet = new HashSet<>();
```

Comment:
wydaje mi się, że w tej wersji z metodami to robiliśmy to zajęciach w oparciu o hashmapę i dlatego wspominaliśmy wtedy o metodach